binding.gyp

```diff
@@ -24,12 +21,19 @@
             "conditions": [
                 [ "SASL==\"n\"", { "sources!": 
                   ["LDAPSASL.cc", "SASLDefaults.cc"] } ], 
-                [ "SASL==\"y\"", { "sources!": ["LDAPXSASL.cc"] } ]
+                [ "SASL==\"y\"", { "sources!": ["LDAPXSASL.cc"] } ],
+		['OS=="linux" and NODE_VERSION > 9', {
+		    "libraries": [ "../deps/libldap.a", "../deps/liblber.a", "-lresolv", "-lsasl2" ],
```

Comment:
use only for RH7 or CentOS 7

Reply:
and how is libldap.a and the other .a file built? If its something you had to manually do, we should have the method of building it documented or automated in this repo

Reply:
we need to support RH6, so should definitely contain a binary for that

Reply:
I have  it automated on napi-ldap you just set the BUILD_OPENLDAP environment variable to 1 and then run yarn and it'll build openssl and openldap for you, see here https://github.com/jjhoughton/napi-ldap/blob/master/binding.gyp. I could either add the same functionality to this project or just link to it from here. That's basically what i did just copy the .a files from the napi-ldap project.

Reply:
I'm a little reluctant to put a great deal of effort into this project as i've already fixed so many of the issues with it in napi-ldap. Especially as jeremy childs isn't to responsive when it comes to accepting patches.